src/pb_utils.h

```diff
@@ -76,34 +76,40 @@ namespace bi = boost::interprocess;
     }                                                                     \
   } while (false)
 
+#define THROW_IF_ERROR(MSG, X)           \
+  do {                                   \
+    int return__ = (X);                  \
+    if (return__ != 0) {                 \
+      throw PythonBackendException(MSG); \
+    }                                    \
+  } while (false)
 
-struct InitializeResponse {
+struct InitializeResponseShm {
```

Comment:
Are these Shm renaming required?

Reply:
It is not required just a new convention I'm trying to follow. All the structs that reside in shared memory are suffixed with `Shm`.